acquia_lift.module

```diff
@@ -0,0 +1,262 @@
+      $output .= '<p>' . t('You can find more info in <a href="@lift-documentation" target="_blank">Documentation</a>.', array(
+          '@lift-documentation' => 'https://docs.acquia.com/lift'
+        ));
+      return $output;
+  }
+}
+
+/**
+ * Returns whether or not Acquia Lift has been configured.
+ *
+ * @return bool
+ *   TRUE if all relevant configuration settings have been set, FALSE
+ *   otherwise.
+ */
+function acquia_lift_is_configured() {
+  foreach (array('acquia_lift_account_id','acquia_lift_site_id', 'acquia_lift_assets_url') as $api_config) {
+    $val = variable_get($api_config, '');
+    if (empty($val)) {
+      return FALSE;
+    }
+  }
+  return TRUE;
+}
+
+/**
+ * Implements hook_page_build().
+ */
+function acquia_lift_page_build(&$page) {
+  // Bail if Acquia Lift has not yet been configured.
+  if (!acquia_lift_is_configured()) {
+    return;
+  }
+
+  // Bail if the path is a path to ignore
+  $current_path = '/' . current_path();
+  $alias_path = drupal_lookup_path('alias', current_path());
+  $alias_path = $alias_path == FALSE ? $current_path : '/' . $alias_path;
+  $patterns = variable_get('acquia_lift_ignore_path_patterns', '');
+  if (drupal_match_path($current_path, $patterns) || drupal_match_path($alias_path, $patterns)) {
+    return;
+  }
+
+  $assets_url = variable_get('acquia_lift_assets_url');
+  $decision_api_url = variable_get('acquia_lift_decision_api_url', '');
+  $oauth_url = variable_get('acquia_lift_oauth_url', '');
+
+  // Add the credentials and configuration for the Lift Experience Builder.
+  _acquia_lift_add_meta_tag('account_id', variable_get('acquia_lift_account_id'));
+  _acquia_lift_add_meta_tag('site_id', variable_get('acquia_lift_site_id'));
+  _acquia_lift_add_meta_tag('liftAssetsUrl', $assets_url);
+  if (!empty($decision_api_url)) {
+    _acquia_lift_add_meta_tag('liftDecisionAPIURL', variable_get('acquia_lift_decision_api_url'));
+  }
+  if (!empty($oauth_url)) {
+    _acquia_lift_add_meta_tag('authEndpoint', variable_get('acquia_lift_oauth_url'));
+  }
+  _acquia_lift_add_meta_tag('contentReplacementMode', variable_get('acquia_lift_content_replacement_mode', 'untrusted'));
+
+  // If there are querystring parameters present corresponding to the configured
+  // identity params, we add the identity and identity type to the meta tags.
+  $identity_param = variable_get('acquia_lift_identity_param', '');
+  $identity_type_param = variable_get('acquia_lift_identity_type_param', '');
+  if (!empty($identity_param)) {
+    $all_params = drupal_get_query_parameters();
+    if (isset($all_params[$identity_param])) {
+      // Set the identity type to the default either as specified by the user or
+      // as the application default.
+      $identity_type = variable_get('acquia_lift_default_identity_type') ?: ACQUIA_LIFT_DEFAULT_IDENTITY_TYPE_DEFAULT;
+      if (!empty($identity_type_param) && isset($all_params[$identity_type_param])) {
```

Comment:
I think ```!empty``` is better than ```isset``` here, because it guards malformed query string such as ```?identity=shay1117a&identityType```, allowing the identityType to fallback to the default.

The D8 counter-part is updated in LEX-1558.